lstchain/data/filters_trasmission.dat

```diff
@@ -0,0 +1,37 @@
+# filters, tranmission estimated from camera data (June 2021, Y. Kobayashi)
+11 1.00000
+12 0.30000
+13 0.14000
+14 0.07300
+15 0.01800
+16 0.00300
+21 0.35000
+22 0.10500
+23 0.04900
+24 0.02555
+25 0.00630
+26 0.00105
+31 0.11000
+32 0.03300
+33 0.01540
+34 0.00803
+35 0.00198
+36 0.00033
+41 0.03000
+42 0.00900
+43 0.00420
+44 0.00219
+45 0.00054
+46 0.00009
+51 0.00300
+52 0.00090
+53 0.00042
+54 0.00022
+55 0.00005
+56 0.00001
+61 0.00050
+62 0.00015
+63 0.00007
+64 0.00004
+65 0.00001
+66 0.00000
```

Comment:
exactly 0? Or do we need more digits here?

Reply:
For the camera is practically zero, but I can indeed add some digit